cql3/functions/aggregate_fcts.hh

```diff
@@ -247,6 +247,25 @@ struct aggregate_type_for<timeuuid_native_type> {
     using type = timeuuid_native_type::primary_type;
 };
 
+template<>
+struct aggregate_type_for<time_native_type> {
+    using type = time_native_type::primary_type;
+};
+
+template <typename Type>
+const Type& max_wrapper(const Type& t1, const Type& t2) {
+    using std::max;
+    return max(t1, t2);
```

Comment:
nitpick: why do with two lines what you can do with one, i.e., call "std::max()" directly?

Reply:
Thanks! I use this trick sometimes to allow custom implementations, e.g. `max(const WeirdType& wt1, const WeirdType& wt2)` in the namespace where `WeirdType` belongs. Then ADL will prefer the custom over `std::max()`.
But you're right, at this moment it does the same thing, it's only for future-proofness.